c2h/include/c2h/generators.h

```diff
@@ -45,12 +45,12 @@ _CCCL_DIAG_SUPPRESS_CLANG("-Wunused-function")
 #    include <cuda_bf16.h>
 _CCCL_DIAG_POP
 
-#    if _CCCL_CUDACC_AT_LEAST(11, 8)
+#    if _CCCL_CUDA_COMPILER_AT_LEAST(11, 8)
```

Comment:
We cannot change that macro, This relates to the `__CUDACC__` macro, aka the version of the cuda headers used 

This is different from the version of e.g clang-cuda and also nvc++ can consume cuda headers from different versions.

So we should leave that macro alone

Reply:
@bernhardmgruber @gevtushenko Do you have a opinions here?

Reply:
@miscco it seems you already have a far better understanding what `__CUDACC__` means, so I leave this decision to you.

I agree that if `__CUDACC__` does not designate a CUDA compiler version, then renaming `_CCCL_CUDACC_AT_LEAST` to `_CCCL_CUDA_COMPILER_AT_LEAST` is misleading.